e2e/tests/nightly/admin/user-profile-admin.spec.ts

```diff
@@ -0,0 +1,469 @@
+    } else {
+      expect(newStatus).toBe('Bypassed');
+      expect(await userProfileAdminPage.getDataAccessTiers()).toEqual('Registered Tier');
+    }
+
+    if (newStatus === 'Incomplete') {
+      await bypassSwitch.turnOn();
+      await saveButton.click();
+      await userProfileAdminPage.waitForLoad();
+    }
+  });
+
+  test('Can bypass Controlled Tier training module', async () => {
+    const userProfileAdminPage = new UserProfileAdminPage(adminTab);
+    await userProfileAdminPage.waitForLoad();
+
+    const accessStatusTable = await userProfileAdminPage.getAccessStatusTable();
+    const expiresOnCell = await accessStatusTable.getCellByValue(AccessModules.CT_TRAINING, TableColumns.EXPIRES_ON);
+    const expiresOn = await expiresOnCell.getCellValue();
+    expect(expiresOn).toEqual('-');
+
+    // Change Bypass for Google 2-step verification module
+    const statusCell = await accessStatusTable.getCellByValue(AccessModules.CT_TRAINING, TableColumns.STATUS);
+    const oldStatus = await statusCell.getCellValue();
+
+    const bypassSwitch = await userProfileAdminPage.getBypassSwitchForRow(AccessModules.CT_TRAINING);
+    if (oldStatus === 'Bypassed') {
+      expect(await userProfileAdminPage.getDataAccessTiers()).toEqual('Registered Tier');
+      await bypassSwitch.turnOff();
+    } else {
+      expect(await userProfileAdminPage.getDataAccessTiers()).toEqual('Registered Tier');
+      await bypassSwitch.turnOn();
+    }
+
+    const bypassSwitchCell = await accessStatusTable.getCellByValue(AccessModules.CT_TRAINING, 'Bypass');
+    await verifyToggleSwitchNewBackgroundColor(adminTab, bypassSwitchCell);
+
+    const saveButton = userProfileAdminPage.getSaveButton();
+    await verifyButtonEnabled(saveButton, true);
+    await saveButton.click();
+    await userProfileAdminPage.waitForLoad();
+
+    const newStatus = await statusCell.getCellValue();
+    if (oldStatus === 'Bypassed') {
+      expect(newStatus).toBe('Incomplete');
+      expect(await userProfileAdminPage.getDataAccessTiers()).toEqual('Registered Tier');
+    } else {
+      expect(newStatus).toBe('Bypassed');
+      expect(await userProfileAdminPage.getDataAccessTiers()).toEqual('Registered Tier');
+    }
+
+    if (newStatus === 'Incomplete') {
+      await bypassSwitch.turnOn();
+      await saveButton.click();
+      await userProfileAdminPage.waitForLoad();
+    }
+  });
+
+  test('Can modify editable information', async () => {
+    const userProfileAdminPage = new UserProfileAdminPage(adminTab);
+    await userProfileAdminPage.waitForLoad();
+
+    // Modify Institution Role
+    const institutionRole = userProfileAdminPage.getInstitutionalRole();
+    const oldRole = await institutionRole.getSelectedValue();
+
+    let options = (await institutionRole.getAllOptionTexts()).filter((role) => role !== oldRole);
+    const randRole = fp.shuffle(options)[0];
+    await institutionRole.select(randRole);
```

Comment:
careful here - if it chooses Other, then you have to enter the Other Text also.  May want to remove that choice.

Reply:
great!  Could you add a comment explaining why we did this?